cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js

```diff
@@ -1,5 +1,9 @@
 angular.module(PKG.name + '.feature.admin').controller('AdminNamespaceDatasetMetadataController',
+<<<<<<< HEAD
```

Comment:
The previous merge got some issue. Probably due to a pull again. Anyway, it's fixed.